### PR TITLE
[4.0] Drop inline script for multiselect.js

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue # .

### Summary of Changes
We don't need inline scripts

### Testing Instructions

List views continue to work as before

### CODE REVIEW

File: media/system/js/multiselect.js [hunk 1]
```diff
@@ -8,87 +8,101 @@
...
-	
-			changeBg(this, cbClicked);
-		});
+					changeBg(this, cbClicked);
+				});
+			});
+		}
 	});
-});
+})();
```
Comment (C-Lodder) [inline media/system/js/multiselect.js]: `})();` >> `})(Joomla);` to fix the console log error

Comment (dgt41) [inline media/system/js/multiselect.js]: ah, grrr

Comment (zero-24) [inline media/system/js/multiselect.js]: hmm this is still `})();`? So this fix is still needed?

**Comment by zero-24** [comment]
@dgt41 looks like here are conflicts too.

**Pull request merged**